aiogram/bot/base.py

```diff
@@ -361,5 +364,22 @@ def disable_web_page_preview(self, value):
     def disable_web_page_preview(self):
         self.disable_web_page_preview = None
 
+    @property
+    def allow_sending_without_reply(self):
+        return getattr(self, '_allow_sending_without_reply', None)
```

Comment:
Something messed with `_allow_sending_without_reply` and `allow_sending_without_reply`.
On set you're modifying `_allow_sending_without_reply`
On delete you're modifying `allow_sending_without_reply`

Reply:
i think it's ok to modify property with `setter`. `setter` understand `None` value

Reply:
> On set you're modifying `_allow_sending_without_reply`
> On delete you're modifying `allow_sending_without_reply`

@Olegt0rr same in global `parse_mode`.

https://github.com/aiogram/aiogram/blob/b3263f9d6ba6eb1ba0c0d62d1d0788853a6faeb5/aiogram/bot/base.py#L323-L345